src/main/java/org/jabref/gui/collab/EntryChangeViewModel.java

```diff
@@ -27,103 +20,44 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(EntryChangeViewModel.class);
 
-    private final List<FieldChangeViewModel> fieldChanges = new ArrayList<>();
+    private final BibEntry oldEntry;
+    private final BibEntry newEntry;
+    private MergeEntries mergePanel;
+
+    private final BibDatabaseContext database;
 
-    public EntryChangeViewModel(BibEntry entry, BibEntry newEntry) {
+    public EntryChangeViewModel(BibEntry entry, BibEntry newEntry, BibDatabaseContext database) {
         super();
+
+        this.oldEntry = entry;
+        this.newEntry = newEntry;
+        this.database = database;
+
         name = entry.getCiteKeyOptional()
                     .map(key -> Localization.lang("Modified entry") + ": '" + key + '\'')
                     .orElse(Localization.lang("Modified entry"));
 
-        Set<Field> allFields = new TreeSet<>(Comparator.comparing(Field::getName));
-        allFields.addAll(entry.getFields());
-        allFields.addAll(newEntry.getFields());
-
-        for (Field field : allFields) {
-            Optional<String> value = entry.getField(field);
-            Optional<String> newValue = newEntry.getField(field);
-
-            if (value.isPresent() && newValue.isPresent()) {
-                if (!value.equals(newValue)) {
-                    // Modified externally.
-                    fieldChanges.add(new FieldChangeViewModel(entry, field, value.get(), newValue.get()));
-                }
-            } else {
-                // Added/removed externally.
-                fieldChanges.add(new FieldChangeViewModel(entry, field, value.orElse(null), newValue.orElse(null)));
-            }
-        }
     }
 
     @Override
     public void makeChange(BibDatabaseContext database, NamedCompound undoEdit) {
-        for (DatabaseChangeViewModel c : fieldChanges) {
-            if (c.isAccepted()) {
-                c.makeChange(database, undoEdit);
-            }
-        }
+        database.getDatabase().removeEntry(oldEntry);
+        database.getDatabase().insertEntry(mergePanel.getMergeEntry());
+        undoEdit.addEdit(new UndoableInsertEntry(database.getDatabase(), oldEntry));
+        undoEdit.addEdit(new UndoableInsertEntry(database.getDatabase(), mergePanel.getMergeEntry()));
     }
 
     @Override
     public Node description() {
+
+        mergePanel = new MergeEntries(oldEntry, newEntry, Localization.lang("In JabRef"), Localization.lang("On disk"), DefaultRadioButtonSelectionMode.RIGHT);
```

Comment:
I think by default changes in JabRef have priority over changes on disk (i.e. the checkbox "accept change" is disabled by default). In line with  this convention, it should be `DefaultRadioButtonSelectionMode.LEFT` here.

Reply:
I'd say, the changes on disk have priority. OK, it depends on the use case. I played with it as follows: Opened lib in JabRef. Openeded lib in Notepad++. Changed file in Notepad++. Thus, I expected the Notepad++ changes having priority. -- In case a user doesn't save often, the changes by JabRef have priority. Users saving often would IMHO expect the changes on disk having priority. Thus, I voted for that.

Reply:
I don't have a strong opinion on this one, it should be consistent however.

Reply:
Just though a bit: in case the user changed something in JabRef, this should be kept as default. Therefore, default should be JabRef.

Reply:
Default is now lef